src/components/locale/cultures/la-IT.js

```diff
@@ -22,7 +22,9 @@ Soho.Locale.addCulture('la-IT', {
       dayOfWeek: 'EEE d',
       timestamp: 'HH:mm:ss',
       hour: 'HH:mm',
-      datetime: 'M/d/yyyy HH:mm'
+      datetime: 'M/d/yyyy HH:mm',
```

Comment:
Hi, I noticed that this part does not match the short date format of la-IT. i didn't know how to extend to all three lines after adding the comment, so it became 2 comments. I hope that is OK.
```suggestion
      datetime: 'dd/MM/yyyy HH:mm',
```